### DISCUSSION TITLE
Trouble using xpublish for kerchunk files. ValueError: Encoding chunks do not match inferred chunks

### DISCUSSION THREAD

**Discussion by noaaroland** [xarray-contrib/xpublish#276] [Q&A]
I have successfully used xpublish for similar model output. This is a different model and the files are organized with the grid spec and data in separate files. I expected xpublish to work similarly as the other model. When I kerchunk the files and read the MultiZarrToZarr kerchunk files (read the time-aggregated data and merge it with the grid) directly everything works fine and I have a CF xarray data set I can use.

However, when I serve the kerchunk files I consistently get the ValueError. I've thrashed around trying a bunch of different things, but nothing I tired changed the error. I'm hoping I'm doing something wrong that y'all can spot from the example.

FWIW, here's an example of what I'm talking about: https://github.com/NOAA-PMEL/GoA_xpublish

```
xpublish                  0.3.3              pyhd8ed1ab_0    conda-forge
cf_xarray                 0.9.2              pyhd8ed1ab_0    conda-forge
xarray                    2024.6.0           pyhd8ed1ab_1    conda-forge 
```

**Comment by jhamman** [comment]
@noaaroland - which `ValueError` are you referring to. I'm not following.

**Reply by noaaroland** [reply]
The one I mention in the title:

ValueError: Encoding chunks do not match inferred chunks
